File: C++/program19.cpp

#include<iostream>
using namespace std;


bool DivisbleByFiveThree(int iNo)
{
    bool bflag =  false;

    if(((iNo % 3) == 0) && ((iNo % 5) == 0))
    {
        bflag = true;
    }
    
    return  bflag;
}

int main()
{
    int iValue = 0;
    bool bRet = false;

    cout<<"Enter number to check the number is divisible by 3 and 5 or not"<<'\n';
    cin>>iValue;

    bRet = DivisbleByFiveThree(iValue);

    if(bRet == true)
    {
        cout<<"Divisble by 3 and 5 "<<'\n';
    }
    else
    {
        cout<<"Not divisible 3 and 5  "<<'\n';
    }

    return 0;
}  

File: Java/program19.java
import java.util.*;

class Even
{
    public boolean CheckEven(int iNo)
    {
        if((iNo % 2) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
}   }

class program19
{
    public static void main(String A[])
    {
        int iValue = 0;
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number :");
        iValue = sobj.nextInt();

        Even eobj = new Even();
        bRet = eobj.CheckEven(iValue);

        if(bRet == true)
        {
            System.out.println(iValue +" is Even number");

        }
        else
        {
            System.out.println(iValue+" is Odd number  ");
        }

               

        
    }
}